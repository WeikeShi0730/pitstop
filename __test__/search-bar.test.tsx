import { render, RenderResult } from "@testing-library/react";
import SearchBar from "../components/search-bar.component";

let documentBody: RenderResult;
describe("<SearchBar />", () => {
  beforeEach(() => (documentBody = render(<SearchBar />)));
  it("Test <SearchBar />", () => {
    const { baseElement } = documentBody;
    expect(baseElement).toMatchSnapshot();
  });
});
