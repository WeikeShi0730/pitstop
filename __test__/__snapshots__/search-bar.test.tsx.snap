// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> Test <SearchBar /> 1`] = `
<body>
  <div>
    <div
      class="flex justify-center items-center gap-2 overflow-hidden text-sm md:text-base"
    >
      <button
        class="flex font-light justify-center items-center py-2"
        id="searchBtn"
      >
        <svg
          fill="none"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="11"
            cy="11"
            r="8"
          />
          <line
            x1="21"
            x2="16.65"
            y1="21"
            y2="16.65"
          />
        </svg>
      </button>
    </div>
  </div>
</body>
`;
